### ISSUE TITLE
Cura AppImage shows no icon

### ISSUE THREAD

**Issue by probonopd** [Ultimaker/cura-build#201]
Unlike other AppImages, the Cura one is missing an icon (when using with the optional `appimaged` daemon that integrates AppImages into the system).

![icons](https://user-images.githubusercontent.com/2480569/48312455-a3c92380-e5a6-11e8-906d-f25b1f5ef380.png)

An icon should (also) be provided in `usr/share/icons/hicolor/256x256/apps/cura.png` to help the optional `appimaged` daemon and tools like it.

**Comment by probonopd** [comment]
Similarly in the menu (when using with the optional `appimaged` daemon that integrates AppImages into the system):

![peek 2018-11-11 11-50](https://user-images.githubusercontent.com/2480569/48312548-5483f280-e5a8-11e8-9cd1-ee6748674d28.gif)

Screencast recorded with [Peek AppImage](https://github.com/phw/peek/releases/).

**Comment by probonopd** [comment]
Please copy the icon to the location defined by the freedesktop spec before runnnig `appimagetool`.

https://github.com/Ultimaker/cura-build/blob/e5e52c0a171ccae0319607667408c968f402eddd/packaging/linux.cmake#L69

Something like

```
COMMAND mkdir -p usr/share/icons/hicolor/256x256/apps/ && cp cura-icon.png usr/share/icons/hicolor/256x256/apps/
```

might do.

**Comment by LipuFei** [comment]
Fixed in 1d4b87b
